src/main/java/uk/gov/hmcts/reform/pip/account/management/service/MediaApplicationService.java

```diff
@@ -178,6 +178,22 @@ public void deleteApplication(UUID id) {
         mediaApplicationRepository.delete(applicationToDelete);
     }
 
+    public String deleteAllApplicationsWithEmailPrefix(String prefix) {
+        List<MediaApplication> applicationsToDelete = mediaApplicationRepository
+            .findAllByEmailStartingWithIgnoreCase(prefix);
+
+        if (!applicationsToDelete.isEmpty()) {
+            applicationsToDelete.forEach(a -> azureBlobService.deleteBlob(a.getImage()));
```

Comment:
If the application has been approved / rejected, the file will no longer exist and I think will cause an error here.

I think we'll have to add a check to capture that the file is still present